test/main.pony

```diff
@@ -35,6 +35,8 @@ class _TestParsePlatform is UnitTest
         ("ponyc-?-?-x86_64-alpine-linux-musl", (AMD64, Linux, "musl"))
         ("?-?-?-x86_64-alpine-linux-musl", (AMD64, Linux, None))
         ("ponyc-?-?-x86_64-apple-darwin", (AMD64, Darwin, None))
+        ("ponyc-?-?-x86_64-freebsd", (AMD64, FreeBSD, None))
+        ("?-?-?-x86_64-freebsd", (AMD64, FreeBSD, None))
         ("?-?-?-darwin", (AMD64, Darwin, None))
         ( "ponyc-?-?-musl"
         , ( AMD64
```

Comment:
This test and the one below are failing due to the `if Platform.osx() then Darwin else Linux end` without a check for the current platform being FreeBSD. We should add the following utility function for this UnitTest for use in these tests that utilize system detection to fill in defaults:
```pony
fun platform_os(): OS
fun platform_distro(String): Distro
```